releases.yml

```diff
@@ -1,4 +1,39 @@
 releases:
+  ec.1:
+    assembly:
+      basis:
+        brew_event: 51500074
+        reference_releases:
+          aarch64: 4.14.0-0.nightly-arm64-2023-05-20-004425
+          ppc64le: 4.14.0-0.nightly-ppc64le-2023-05-20-010750
+          s390x: 4.14.0-0.nightly-s390x-2023-05-19-225858
+          x86_64: 4.14.0-0.nightly-2023-05-19-234822
+      group:
+        upgrades: 4.13.0-rc.3,4.13.0-rc.4,4.13.0-rc.5,4.13.0-rc.6,4.13.0-rc.7,4.13.0-rc.8,4.13.0,4.14.0-ec.0
```

Comment:
```suggestion
        upgrades: 4.13.0-rc.3,4.13.0-rc.4,4.13.0-rc.5,4.13.0-rc.6,4.13.0-rc.7,4.13.0-rc.8,4.13.0,4.13.1,4.14.0-ec.0
```

Reply:
I didn't set `IN_FLIGHT_PREV` as I thought 4.13.1 would be included since it's already on RC. But it's not accepted yet, so we need to add this upgrade edge by hand